apix-impl/src/main/java/eu/xenit/apix/alfresco/versionhistory/VersionHistoryService.java

```diff
@@ -15,12 +15,15 @@
 import java.util.Map;
 import org.alfresco.service.cmr.repository.NodeService;
 import org.alfresco.service.cmr.version.VersionType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 @Service("eu.xenit.apix.versionhistory.VersionHistoryService")
 public class VersionHistoryService implements IVersionHistoryService {
 
+    private static final Logger log = LoggerFactory.getLogger(VersionHistoryService.class);
```

Comment:
Don't we have the `@Slf4j` annotation?

Reply:
The logger is defined, but it's not used?

Reply:
cleaned up,